Moving onedrive solution inline (#1480)
swcarpentry/shell-novice: episodes/02-filedir.md

this command will return an error. Typically, a `Desktop` directory exists in your
home directory, which we assume is the current working directory of your bash shell.

:::::::::::::: callout

### OneDrive Variations

If a Windows machine is backed up with OneDrive, your Desktop may

>   be rendered from a folder within OneDrive, which will not match the contents of `~/Desktop`.

Author: ```suggestion
  have been moved within OneDrive and you will get an error when you try `ls -F Desktop`.
```

>   The OneDrive desktop should be accessible using one of the following commands
>   (if the name of the enterprise/org isn't clear, look through the output of `ls` to find
>   the right folder):
>   
>   ```bash
>   $ cd "~/OneDrive - Name Of Enterprise/Desktop"
>   $ cd "C:/Users/Username/OneDrive - Name Of Enterprise/Desktop"
>   ```

Author: ```suggestion
  Run the command `ls` and examine the output, looking for a folder that has "OneDrive" in the name and note that folder name. It will likely include the name of your organization in it.  It may be something like `OneDrive - University of Example`
  Going forward, whenever the lesson or instructor uses the `Desktop` folder, you will need to include the OneDrive folder as well. For example if the instructor types...

``` bash
ls -F Desktop
```

You will need to type:

```bash
ls -F "OneDrive - Name Of Enterprise/Desktop"
```

Author: If the language in the above looks okay, I'll fix it not in a github comment.  It isn't interpreting my suggestion correctly

> Note: The quotation marks are needed since the Unix shell interprets spaces to be meaningful separators between files/commands/options.

Author: ```suggestion
Note: The quotation marks are needed for this folder because the Unix shell interprets spaces to be meaningful separators between files/commands/options but this folder has spaces in the name.
```

this command will return an error. Typically, a `Desktop` directory exists in your
home directory, which we assume is the current working directory of your bash shell.

:::::::::::::: callout

### OneDrive Variations

If a Windows machine is backed up with OneDrive, your Desktop may
have been moved within OneDrive and you will get an error when you try `ls -F Desktop`.

To check if this is true for you, run the command `ls` and look through the output for a folder that has "OneDrive" in the name and note that folder name. It will likely include the name of your organization in it.  It may be something like `OneDrive - University of Example`. 
  
If you don't see a "OneDrive" folder, you may not be in your home folder or there may be another issue going on.  To check you are in your home folder, type `pwd` and it should show something like `C:/Users/Username` or `/c/Users/Username`.

If you do see the "OneDrive" folder, going forward whenever the lesson or instructor uses the `Desktop` folder, you will need to include the OneDrive folder as well. For example if the instructor types...

``` bash
ls -F Desktop
```

You will need to type:

```bash

> ls -F "OneDrive - Name Of Enterprise/Desktop"

Author: ```suggestion
ls -F "OneDrive - University of Example/Desktop"
```
